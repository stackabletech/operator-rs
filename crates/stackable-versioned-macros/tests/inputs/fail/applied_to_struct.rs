use stackable_versioned_macros::versioned;

#[versioned()]
pub struct Foo {}

fn main() {}
