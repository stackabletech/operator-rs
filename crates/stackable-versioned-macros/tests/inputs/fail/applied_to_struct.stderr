error: attribute macro `versioned` can be only be applied to modules
 --> tests/inputs/fail/applied_to_struct.rs:4:1
  |
4 | pub struct Foo {}
  | ^^^
